### PR TITLE
NMS-12653: Remove remote poller

### PR DESCRIPTION
**Pull request by fooker**
Remove remote poller GUI, packaging and backend.

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12653

### CODE REVIEW

**Comment by fooker** [comment]
@RangerRick can you have a look at the packaging and the .spec files?

**Comment by RangerRick** [comment]
The RPM spec changes look fine.  I've removed the debian remote poller stuff, looks like there's still a lot to remove.  `system-report` references, `-Dpoller.logfile`, probably basically anything that shows in `git grep -i remoting`...

**Comment by fooker** [comment]
Thanks for the Debian stuff. This is not complete right now. I just wanted to get your feedback because packaging feels like black magic to me...

File: opennms-base-assembly/src/main/filtered/etc/events/opennms.correlation.events.xml [hunk 1]
```diff
@@ -1,26 +1,4 @@
 <events xmlns="http://xmlns.opennms.org/xsd/eventconf">
-   <event>
```
Comment (christianpape) [inline opennms-base-assembly/src/main/filtered/etc/events/opennms.correlation.events.xml:2]: Don't know. Maybe we want to keep such kind of event?

File: opennms-config/src/test/resources/org/opennms/netmgt/config/eventd/eventconf-speedtest/eventconf.xml [hunk 2]
```diff
@@ -1129,160 +1129,6 @@
     <alarm-data reduction-key="%uei%:%dpname%:%nodeid%:%interface%:%service%" alarm-type="3" auto-clean="false" />
 -->
   </event>
-  <!-- Remote Poller Events -->
```
Comment (christianpape) [inline opennms-config/src/test/resources/org/opennms/netmgt/config/eventd/eventconf-speedtest/eventconf.xml:1132]: Same here. Should we keep different events for remote outages?

File: opennms-base-assembly/src/main/filtered/etc/events/opennms.remote.poller.events.xml [hunk 3]
```diff
@@ -1,59 +0,0 @@
-<events xmlns="http://xmlns.opennms.org/xsd/eventconf">
```
Comment (christianpape) [inline opennms-base-assembly/src/main/filtered/etc/events/opennms.remote.poller.events.xml:1]: Same here. Should we keep separate events for remote outages?

File: opennms-web-api/src/main/java/org/opennms/web/svclayer/model/LocationMonitorListModel.java [hunk 4]
```diff
@@ -162,10 +157,7 @@ public int compare(Entry<String, String> one, Entry<String, String> two) {
             });
             for (Entry<String, String> detail : details) {
                 if (
-                    !detail.getKey().equals(PollerBackEnd.HOST_NAME_KEY) &&
```
Comment (christianpape) [inline opennms-web-api/src/main/java/org/opennms/web/svclayer/model/LocationMonitorListModel.java:165]: Wait! What?

Comment (fooker) [reply opennms-web-api/src/main/java/org/opennms/web/svclayer/model/LocationMonitorListModel.java:165]: It's just to get it working. Have to revisit this code anyways.

File: opennms-base-assembly/src/main/filtered/etc/events/opennms.correlation.events.xml (hunk 1 above)
Comment (fooker) [reply opennms-base-assembly/src/main/filtered/etc/events/opennms.correlation.events.xml:2]: Then we will recreate it...

File: opennms-base-assembly/src/main/filtered/etc/events/opennms.remote.poller.events.xml (hunk 3 above)
Comment (fooker) [reply opennms-base-assembly/src/main/filtered/etc/events/opennms.remote.poller.events.xml:1]: Then we will recreate it...

**Pull request merged**